pkg/solana/txm/txm_internal_test.go

```diff
@@ -163,6 +163,12 @@ func TestTxm(t *testing.T) {
 
 			// happy path (send => simulate success => tx: nil => tx: processed => tx: confirmed => finalized => done)
 			t.Run("happyPath", func(t *testing.T) {
+				mc.On("LatestBlockhash", mock.Anything).Return(&rpc.GetLatestBlockhashResult{
+					Value: &rpc.LatestBlockhashResult{
+						LastValidBlockHeight: 100,
+						Blockhash:            solana.Hash{},
+					},
+				}, nil).Once()
```

Comment:
Looks like we mock the `LatestBlockhash` the same way for all of the tests. Could we move this one layer higher so we only have to mock it once? We can ditch the `.Once()` unless some tests don't get that far then I'm ok with replacing it with `.Maybe()`